extension/oauth2clientcredentialsauthextension/README.md

```diff
@@ -0,0 +1,53 @@
+# Authenticator - OAuth2 Client Credentials
+
+This extension implements a `configauth.ClientAuthenticator` and is to be used by HTTP and gRPC based exporters.
+The authenticator type has to be set to `oauth2`.
```

Comment:
I think the type is currently `oauth2clientcredentials` (but see my comment later).

Reply:
Updated the docs.